.travis.yml

```diff
@@ -15,8 +15,12 @@
 
 dist: bionic
 language: java
-jdk: oraclejdk8
-arch: arm64
+jobs:
+    include:
+      - arch: s390x
+        jdk: openjdk11
```

Comment:
any reason to use JDK 11 for s390x ?

Reply:
JDK8 is very slow for s390x and causes test case failures due to timeout.

Reply:
My concern is that Tomcat (even version 10.x) is supposed to be build with JDK 8. 
But since Travis is not the main CI I think it is OK to use JDK 11 as an exception here.